modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,499 @@
+addresses           : ["10.0.54.26/32", "10.0.56.38/32", "10.0.69.31/32", "172.30.0.1/32"]
+external_ids        : {direction=Egress, gress-index="3", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:3:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a10622494091691694581
+
+_uuid               : 77554661-34c7-4d24-82e2-819c9be98e55
+addresses           : ["10.244.1.3", "10.244.2.7"]
+external_ids        : {direction=Egress, gress-index="4", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:4:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a5972452606168369118
+
+_uuid               : c8b87fe8-dd6c-4ef3-8cea-e7ea8a5bdb3d
+addresses           : ["0.0.0.0/0"]
+external_ids        : {direction=Egress, gress-index="5", ip-family=v4, "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:5:v4", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy}
+name                : a11452480169090787059
+----
+====
+
+To troubleshoot a specific policy and rule for that ANP, follow these steps:
+
+. Examine the ACL list for that policy by running the following command:
++
+[source,terminal]
+----
+$ ovn-nbctl list acl | grep <policy_name>
+----
+
+. Examine the specific rule with your policy by running the following command:
++
+[source,terminal]
+----
+$ ovn-nbctl list acl | grep <anp_name>:<direction>:<rule_number>
+----
++
+Where::
++
+`anp_name`:: Specifies the name of your ANP.
+`direction`:: Specifies the direction of traffic either `Ingress` or `Egress`.
+`rule_number`:: Specifies the rule you wish to look at.
++
+For the example ANP named `cluster-control` at `priority` `34`, the following is an example output for `Egress` `rule` 5:
++
+. Example output
+[source,terminal]
+----
+_uuid               : 924b81fd-65cc-494b-9c29-0c5287d69bb1
+action              : drop
+direction           : from-lport
+external_ids        : {direction=Egress, gress-index="5", "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:5:None", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy, port-policy-protocol=None}
+label               : 0
+log                 : false
+match               : "inport == @a14645450421485494999 && ((ip4.dst == $a11452480169090787059))"
+meter               : acl-logging
+name                : "ANP:cluster-control:Egress:5"
+options             : {apply-after-lb="true"}
+priority            : 26595
+severity            : []
+tier                : 1
+----
```

Comment:
This output would be possible only if you removed `grep` as per my suggestion from upstream above. With grep, you would need some `-A`, `-B` or `-C` parameters to get this output.